client/api/notebook.py

```diff
@@ -97,6 +101,21 @@ def save_notebook(self):
         else:
             print()
 
+def login_with_env(assignment):
+    access_token = os.environ.get('OKPY_ACCESS_TOKEN')
+    if not access_token:
+        log.info("ACCESS_TOKEN did not exist in the environment")
+        return
+    student_email = ok_auth.display_student_email(assignment.cmd_args, access_token)
+    if student_email:
+        # Token is valid,
+        expires_in = int(os.environ.get('OKPY_EXPIRES_IN', 60))
+        refresh_token = os.environ.get('OKPY_REFRESH_TOKEN')
```

Comment:
The notebook environment sets this environment variable if the student logins in with OK.